portal-api/src/drivers/fileshare.js

```diff
@@ -100,7 +114,7 @@ const uploadFile = async ({ fileshare, folder, filename, buffer, allowOverwrite
 
   const fileClient = await directoryClient.getFileClient(`${filename}`);
 
-  const existingFileProps = await fileClient.getProperties().catch(() => {});
+  const existingFileProps = await fileClient.getProperties().catch(() => { });
```

Comment:
When we get round to running prettier on everything I think it would remove the added space here and below.